python_modules/dagster/dagster_tests/execution_tests/misc_execution_tests/test_input_values_from_intermediates.py

```diff
@@ -22,19 +22,19 @@ def pipe():
 
     assert result
     assert result.success
-    step_input_event = next(
+    step_input_data = next(
```

Comment:
are these changes to the unit tests related to the `@record` conversion or general cleanup? i think they are fine either way, mostly curious for my own understanding

Reply:
the positional indexing in at [1] (using [1]) is no longer allowed with `@record` so that I needed to fix and the rest is clean up